tests/tests/swfs/avm2/cryptscore/Test.as

```diff
@@ -0,0 +1,28 @@
+﻿package  {
+	
+	import flash.display.MovieClip;
+	import flash.events.Event;
+	
+	
+	
+	public class Test extends MovieClip {
+	
+		
+		public function Test() {
+			var score = new CryptScore();
+			for (var i = 0; i < 10000; i ++) {
+				if (i % 1000 == 0) {
+					trace("i = " + i);
+				}
+				score.value = score.value;
```

Comment:
What does this line do? :thinking:

Reply:
This runs both the getter and setter.

Reply:
Ooh, right!

Reply:
Well at any rate, it wasn't immediately obvious...